mmseg/models/decode_heads/dest_head.py

```diff
@@ -1,53 +1,54 @@
 import torch
 import torch.nn as nn
 
+from mmcv.cnn import ConvModule
 from mmseg.models.builder import HEADS
 from mmseg.models.decode_heads.decode_head import BaseDecodeHead
 
 
 @HEADS.register_module()
 class DestHead(BaseDecodeHead):

```

Comment:
DESTHead? DEST is abbreviation like DW.

Reply:
this is adopted from our open sourced one, any suggestion for name of this head?
btw, what's DW

Reply:
DW is depth-wise, so DestHead can be DESTHead